packages/user-mgnt/src/utils/userUtils.ts

```diff
@@ -24,7 +24,7 @@ interface IRoleScopeMapping {
 
 export const roleScopeMapping: IRoleScopeMapping = {
   FIELD_AGENT: ['declare'],
-  REGISTRATION_AGENT: ['validate'],
+  REGISTRATION_AGENT: ['validate', 'performance', 'certify'],
```

Comment:
I cant remember.  If we give registration agent "performance" scope then are we sure that they cant see the audit reason for user audit?